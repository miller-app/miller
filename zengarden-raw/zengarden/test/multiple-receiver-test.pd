#N canvas 391 203 413 149 10;
#X obj 20 12 loadbang;
#X obj 20 38 s test;
#X obj 20 87 r test;
#X obj 20 119 print test1;
#X obj 105 87 r test;
#X obj 105 119 print test2;
#X connect 0 0 1 0;
#X connect 2 0 3 0;
#X connect 4 0 5 0;
