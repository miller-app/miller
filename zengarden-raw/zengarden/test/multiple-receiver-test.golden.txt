[@ 0.000ms] test1: bang
[@ 0.000ms] test2: bang